bin/resources/GameIndex.yaml

```diff
@@ -19883,10 +19883,10 @@ SLPM-55208:
   name: "Hakuouki - Zuisouroku"
   region: "NTSC-J"
 SLPM-55221:
-  name: "Beatmania IIDX 16 - DJ Empress"
+  name: "beatmania IIDX 16 EMPRESS + PREMIUM BEST (EMPRESS DISC)"
   region: "NTSC-J"
 SLPM-55222:
-  name: "Beatmania IIDX 16 - DJ Empress Premium Best"
+  name: "beatmania IIDX 16 EMPRESS + PREMIUM BEST (PREMIUM BEST DISC)"
```

Comment:
The individual discs have the "EMPRESS DISC" and "PREMIUM BEST DISC" subtitles on them in addition to the full game title. Kind of hard to see in this pic but I can take my own pics of the discs if requested.
https://aucview.aucfan.com/yahoo/w93296025/